pkg/container-runtime/dockerInstaller.go

```diff
@@ -0,0 +1,57 @@
+// Copyright © 2022 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package container_runtime
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/sealerio/sealer/pkg/infradriver"
+)
+
+type DockerInstaller struct {
+	Info   Info
+	rootfs string
+	driver infradriver.InfraDriver
+}
+
+func (d *DockerInstaller) InstallOn(hosts []net.IP) (*Info, error) {
+	RemoteChmod := "cd %s  && chmod +x scripts/* && cd scripts && bash init.sh /var/lib/docker %s %s %s %s"
```

Comment:
In my opinion, the parameters of the init.sh script only need to be related to the container runtime installation process. so, `/var/lib/docker`,` DefaultDomain`, `DefaultPort`, Are these parameters unnecessary?